docs/3.0.0-beta.x/getting-started/quick-start-tutorial.md

```diff
@@ -1,363 +1,628 @@
+
+A **Content Type** can be considered a sort of _blueprint_ for the data created. In other words, a **Content Type** is the schema of the data structure.
+
+The next section guides you through the steps needed for each of these above **Content Type** fields.
 
 ::: tip NOTE
 
-More **Restaurant** themed **Content Types** can be seen in the Strapi demo site: [Strapi Foodadvisor](https://foodadvisor.strapi.io/).
+Additional **Restaurant** themed **Content Types** and fields can be seen in the [FoodAdvisor demo site](https://foodadvisor.strapi.io/).
 
 :::
 
-At this point, your project is empty. You need to create a new **Content Type**.
+### The Restaurant Content Type
+
+- Go to the **Content Type Builder** plugin, located in the left menu: Under **PLUGINS**: --> **Content Type Builder**. You are now able to see the three available **Content Types**.
+
+At this point, three Content Types are available `Permission`, `Role`, and `Users`.
+
+![Content Type Dashboard](../assets/getting-started/tutorial/content-type-dashboard.png 'Content Type Dashboard')
 
-Go to the **Content Type Builder** plugin, located in the left menu: **PLUGINS** --> **Content Type Builder**.
+You need to create a new **Content Type** for `Restaurants`.
 
-You will create two content types. You will create a content type called `Restaurant` and another called `Category`.
+1. Complete these steps to **Add a Restaurant Content Type**:
 
-![Create Content Type Screen](../assets/quick-start-detailed/CreateContentTypeScreen.png 'Create Content Type Screen')
+- Click the `+ Add A Content Type` link (under existing **CONTENT TYPES**)
+- Enter a **Name** for your new **Content Type** (call this `restaurant`), and you can write `Restaurant Listings` for the **Description**
+- Click the `Done` button
+
+![Singular Name Entries for Content Type](../assets/getting-started/tutorial/singular-name-entry.png 'Singular Name Entries or Content Type')
 
 ::: tip NOTE
 
-Content Type `name` is always **singular**. For example, `restaurant` not `restaurants`. For example, `category` not `categories`.
+The Content Type **Name** is always **singular**. For example, `restaurant` not `restaurants`.
 
 :::
 
-### The Restaurant Content Type
+2. You are now at the **Field Selection** panel:
+
+You may add your first field, a **String** field for the **Restaurant** name.
+
+![Field Section Panel](../assets/getting-started/tutorial/field-selection-panel.png 'Field Selection Panel')
 
-The Restaurant Content Type will have a **String** field for the restaurant name, and a **Text** field (with a **WYSIWYG editor**) for the restaurant description.
+- Click on the `String` field
+- In the **Name** field, type `name`. This field is the name of the restaurant.
```

Comment:
The image here has a blue background (because of your auto insert)

Reply:
I fixed this and the user image. Thank you.